setup.py

```diff
@@ -4,9 +4,6 @@
 from setuptools import setup, find_packages
 import sys
 
-if sys.version_info < (3, 0):
-    sys.exit('Sorry, python2 is not yet supported. Please use python3.')
-
 pfile = Project(chdir=False).parsed_pipfile
```

Comment:
@cyon1c you inherited some python bad practices that must be fixed: Pipfile, pipenv, and pip for that matter, should not be needed to install this library. 

The goal for our python library is for it be installable using:

- `pip install steem-python`
- `git clone https://github.com/steemit/steem-python && cd steem-python && python2.7 ./setup.py install`
- `pip3 install steem-python`
- `git clone https://github.com/steemit/steem-python && cd steem-python && python3 ./setup.py install`

To assure that our library really supports python 2 and 3, we need a circleci test setup in our circle.yml that installs our lib and runs the test suite in these envs:
- py27
- py33
- py34
- py35
- py36

Reply:
That was me, and it was a deliberate decision, so that we didn't have to maintain two lists of prereqs and they could just live solely in the Pipfile.  It's documented in the README and I'm ok with it.